lex/lexer.ml

```diff
@@ -179,8 +178,10 @@ let lex_char lexbuf =
 
 let skip_space_and_comments lexbuf =
   let str = get_next_buff lexbuf 2 in
-  match (Bytes.get str 0) with
-  | '(' -> if (Bytes.get str 1) = '*'
+  let len = Bytes.length str in
+  if len == 0 then raise Bad_rule
```

Comment:
I think it would be better to stay with `len = 0` here (`==` is physical equality that our interpreter doesn't deal with so well).

Reply:
arg, again... sorry about that ^^'